Makefile

```diff
@@ -21,7 +22,8 @@ run-geth-testnet:
 		--authrpc.addr=0.0.0.0 \
 		--authrpc.jwtsecret=configs/testnet/jwtsecret \
 		--datadir=data/testnet/geth \
-		--syncmode=full
+		--syncmode=full \
+		--networkid=56200
```

Comment:
I am not certain it is necessary to specify this, as syncing seems to be working correctly without it. If I understand correctly the chain ID is stored in the database when the node is initialised with a custom genesis.json